datahub-web-react/src/app/settings/SettingsPage.tsx

```diff
@@ -43,19 +85,48 @@ export const SettingsPage = () => {
                         <PageTitle level={3}>Settings</PageTitle>
                         <Typography.Paragraph type="secondary">Manage your DataHub settings.</Typography.Paragraph>
                     </SettingsBarHeader>
+                    <ThinDivider />
                     <Menu
                         selectable={false}
                         mode="inline"
-                        style={{ width: 256 }}
-                        selectedKeys={[selectedKey]}
-                        onClick={(key) => {
-                            onMenuClick(key);
+                        style={{ width: 256, marginTop: 8 }}
+                        selectedKeys={[activePath]}
+                        onClick={(newPath) => {
+                            history.push(`${url}/${newPath.key}`);
                         }}
                     >
-                        <Menu.Item key="access-tokens">Access Tokens</Menu.Item>
+                        <Menu.ItemGroup title="Developer">
+                            <Menu.Item key="tokens">
+                                <SafetyCertificateOutlined />
+                                <ItemTitle>Access Tokens</ItemTitle>
+                            </Menu.Item>
+                        </Menu.ItemGroup>
+                        {(showPolicies || showUsersGroups) && (
+                            <Menu.ItemGroup title="Access">
+                                {showPolicies && (
+                                    <Menu.Item key="identities">
+                                        <UsergroupAddOutlined />
+                                        <ItemTitle>Users & Groups</ItemTitle>
+                                    </Menu.Item>
+                                )}
+                                {showUsersGroups && (
+                                    <Menu.Item key="policies">
+                                        <BankOutlined />
+                                        <ItemTitle>Privileges</ItemTitle>
+                                    </Menu.Item>
+                                )}
+                            </Menu.ItemGroup>
+                        )}
                     </Menu>
                 </SettingsBarContainer>
-                {selectedKey === 'access-tokens' && <AccessTokens />}
+                <Switch>
+                    <Route exact path={path}>
+                        <Redirect to={`${pathname}${pathname.endsWith('/') ? '' : '/'}${DEFAULT_PATH.path}`} />
+                    </Route>
+                    {PATHS.map((p) => (
+                        <Route path={`${path}/${p.path.replace('/', '')}`} render={() => p.content} key={p.path} />
```

Comment:
why do you have this `replace` logic here if there's no "/" in any of the paths above?